scripts/eurostat/health_determinants/Alcohol_Consumption/process.py

```diff
@@ -0,0 +1,749 @@
+        df['sex']+'_AlcoholConsumption'+\
+        '_AsAFractionOf_Count_Person_'+df['citizen']+'_'+df['sex']
+    # dropping unwanted columns
+    df.drop(columns=['frequenc','citizen','sex','unit','age'],inplace=True)
+    # ask
+    df = df.melt(id_vars=['SV','time'], var_name='geo'\
+        ,value_name='observation')
+    return df
+
+#health_determinant_eurostat_historical_alcoholconsumption_education_attainment
+def hlth_ehis_de10(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Cleans the file hlth_ehis_de10 for concatenation in Final CSV.
+    Arguments: df (pd.DataFrame), the raw df as the input
+    Returns: df (pd.DataFrame), provides the cleaned df as output
+    """
+    # providing column names
+    columns = ['sex,age,isced11,frequenc,unit,time','BE','BG','CZ','EL',
+        'ES','FR','CY','LV','HU','MT','PL','RO','SI','SK','TR']
+    df.columns=columns
+    # spliting the first column to multiple columns
+    split_columns = "sex,age,isced11,frequenc,unit,time"
+    df = _split_column(df,split_columns)
+    # Filtering out the wanted rows and columns.
+    df = df[df['age'] == 'TOTAL']
+    df = _replace_isced11(df)
+    df = _replace_frequenc(df)
+    df = _replace_sex(df)
+    # giving proper statvar name
+    df['SV'] = 'Count_Person_'+df['isced11']+'_'+df['sex']+'_AlcoholConsumption_'+\
+        df['frequenc']+'_AsAFractionOf_Count_Person_'+\
+            df['isced11']+'_'+df['sex']
+    # dropping unwanted columns
+    df.drop(columns=['isced11','sex','age','frequenc','unit'],inplace=True)
+    # ask
+    df = df.melt(id_vars=['SV','time'], var_name='geo'\
+        ,value_name='observation')
+    return df
+
+#health_determinant_eurostat_historical_bingedrinking_education_attainment
+def hlth_ehis_de6(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Cleans the file hlth_ehis_de6 for concatenation in Final CSV.
+    Arguments: df (pd.DataFrame), the raw df as the input
+    Returns: df (pd.DataFrame), provides the cleaned df as output
+    """
+    # providing column names
+    columns = ['frequenc,sex,age,isced11,time','BE','BG','CZ','EL',
+        'ES','CY','LV','HU','MT','RO','SI','SK']
+    df.columns=columns
+    # spliting the first column to multiple columns
+    split_columns = "frequenc,sex,age,isced11,time"
+    df = _split_column(df,split_columns)
+    # Filtering out the wanted rows and columns.
+    df = df[df['age'] == 'TOTAL']
+    df = _replace_isced11(df)
+    df = _replace_frequenc(df)
+    df = _replace_sex(df)
+    # giving proper statvar name
+    df['SV'] = 'Count_Person_'+df['isced11']+'_'+df['sex']+'_BingeDrinking_'+\
+        df['frequenc']+'_AsAFractionOf_Count_Person_'+\
+            df['isced11']+'_'+df['sex']
+    # dropping unwanted columns
+    df.drop(columns=['isced11','sex','age','frequenc'],inplace=True)
+    # ask
+    df = df.melt(id_vars=['SV','time'], var_name='geo'\
+        ,value_name='observation')
+    return df
+
+def _split_column(df: pd.DataFrame,col: str) -> pd.DataFrame:
```

Comment:
Please re-arrange functions (define them before it is being called for first time)